packages/merge-styles/README.md

```diff
@@ -173,12 +177,39 @@ This would register the rules:
 .ms-Fabric.is-focusVisible .css-0 { background: red; }
 .css-0 .child { background: green; }
 ```
+### Referencing child elements within the mergeStyleSets scope
+
+One important concept about `mergeStyleSets` is that it produces a map of class names for the given elements:
+
+```tsx
+mergeStyleSets({
+  root: { background: 'red' }
+  thumb: { background: 'green' }
+});
+```
+
+In some cases, you may need to alter one child area by interacting with the parent. For example, when the parent focus is set, change the child background. You can reference the areas defined in the style set using $ tokens:
+
+```tsx
+mergeStyleSets({
+  root: {
+    selectors: {
+      ':hover $thumb': { background: 'lightgreen' }
+    }
+   }
+  thumb: { background: 'green' }
+});
+```
+
+The `$thumb` reference in the selector on root will be replaced with the class name generated for thumb.
 
 ## Custom class names
 
-By default class names that are generated will use the prefix `css-` followed by a number, creating unique rules where needed.
+By default when using `mergeStyles`, class names that are generated will use the prefix `css-` followed by a number, creating unique rules where needed.
+
+When using `mergeStyleSets`, class names automatically use the area name as the prefix.
```

Comment:
I may figure it out when I review the rest of the code, but it's not clear on reading what this line means.

Reply:
For example:
```
mergeStyles({ background: 'red'); // produces a string 'css-0'

mergeStyleSet({
  root: { background: 'red' },
  stuff: { background: 'green' }
});
// Produces { root: 'root-0', stuff: 'stuff-1' }

Reply:
Yeah, I figured it out from the code. But it would be good to clarify that in the README

Reply:
Refined README docs.